core/commands/repo.go

```diff
@@ -302,13 +302,35 @@ var repoVerifyCmd = &oldcmds.Command{
 
 		var fails int
 		var i int
+		var count int
+		verify := make(chan string)
+		defer close(verify)
+
 		for k := range keys {
-			_, err := bs.Get(k)
-			if err != nil {
+			go func() {
+				_, err := bs.Get(k)
```

Comment:
`k` should be passed in by the `go func(k cid.Cid){...}(k)`

Reply:
Thx a lot.